coderd/workspaceapps/proxy.go

```diff
@@ -573,7 +573,7 @@ func (s *Server) proxyWorkspaceApp(rw http.ResponseWriter, r *http.Request, appT
 	}
 
 	// This strips the session token from a workspace app request.
-	cookieHeaders := r.Header.Values("Cookie")[:]
+	cookieHeaders := r.Header.Values("Cookie")
```

Comment:
Is there a reason we want to avoid creating the new slice here?

Reply:
I have no clue how this change even got in tbh, thanks for the catch

Reply:
Might be an automated linter thing